src/ga_session.cpp

```diff
@@ -1116,6 +1116,7 @@ namespace sdk {
         cleanup_appearance_settings(locker, appearance);
 
         m_earliest_block_time = m_login_data["earliest_key_creation_time"];
+        m_csvtime = m_login_data["csv_blocks"];
```

Comment:
since this is a relative time, I think its best named m_csv_blocks.

Reply:
Renamed.
Should I rename the settings key to `csv_blocks` as well?
It would be inconsistent with the api name `set_csvtime` but it would not be a breaking change.